tools/Makefile.unix

```diff
@@ -31,7 +31,7 @@ endif
 
 # In case we cannot get version information from GIT
 
-ifneq ($(GIT_PRESENT),true)
+ifeq ($(GIT_PRESENT),)
```

Comment:
@anchao I just tested this out of tree and it breaks again,

Reply:
> @anchao I just tested this out of tree and it breaks again,

emm...  In fact, I do not quite understand what happened on PX4 build, I would like to try it later...

Reply:
Ok but gist of it is: a .git FILE exist for a submodule. To build out of tree It is copied out of tree.  There it exist as a .git file not in a repo. Then git commands will not work.  There was a bad assumption in the original scripts that .git was a directory.